Lesson9/merve/index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Time-Saving CSS Techniques to Create Responsive Images</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <body>
+      <div class="first-box">
+        <h1>Time-saving CSS techniques to create responsive images</h1>
+        <div class="social-media-profile">
+          <img class="first-img" src="https://miro.medium.com/v2/resize:fill:44:44/1*p3v_yjsMfK5k1oKYDXij7Q.jpeg"
+    alt="Adrien Zaganelli"/>
+          <img class="second-img" src="https://miro.medium.com/v2/resize:fill:24:24/1*MotlWcSa2n6FrOx3ul89kw.png" alt="freecodecamp logo"/>
+        </div>
+        <div class="profile">
+          <a class="link" href="https://medium.com/@adri_zag">Adrien Zaganelli</a>
+              <a class="btn" rel="noopener follow" href="https://medium.com/m/signin">Follow</a>
+        </div>
+        <div class="next-info">
+           <span class="light">Published in</span>
+              <a class="link" href="https://medium.com/free-code-camp"
+                >We’ve moved to freeCodeCamp.org/news</a>
+              <span class="light"> · 3 min read · Aug 12, 2018</span>
+        </div>
+        <div class="divider"></div>
+        <div class="social-media-box">
+          <div class="left"></div>
+          <div class="right"></div>
+        </div>
+        <div class="divider"></div>
+        </div>
+        <img class="clock" loading="eager" role="presentation" src="https://miro.medium.com/v2/resize:fit:1250/1*aVzJTznRRfP1lM7AXe9yLw.jpeg" alt=""/>
+      <figcaption>Photo by <a class="link" href="">  &nbspKevin Ku</a>&nbspon<a class="link" href=""> &nbspUnsplash</a></figcaption>
+      <main>
+        <div class="article">
+          <p>As a web developer, there is a high probability that you have encountered the two enemies of this article: <b>images</b> and <b>deadlines.</b> Sometimes, for some reasons, your images won’t fit the layout and you don’t want to wrap your head around this for hours.</p>
+    <p>This situation has happened to me many times and I have learned from my mistakes. No more black magic hacks — here are my five favorites techniques to handle image resizing.</p>
+          <h2>the “OMG I NEED THIS ASAP” way</h2>
+    <p>It’s 5:00 pm on Friday, you have to finish this page, but the images won’t fit the layout. It’s time to use your magic trick!</p>
+        <pre>
+    .myImg{
+    background-image: url("my-image.png");
+    background-size: cover;
+    }</pre>
+          <p>Sounds familiar? We’ve all done it once, doesn’t it feel like cheating to you?</p>
+          <p>Using<cite> background</cite> properties is very useful, they just work. Yet, remember that you should use them only for non-content images or as a replacement of text and in <a href="">some particular cases</a>.</p>
```

Comment:
Use <code> element instead of <cite> for code inside the text. In case you have something like angle brackets, you'd need to replace that characters with HTML code value, e.g. :
instead of `<code><img /></code>` you'd have to write `<code>&lt;img&gt;</code>`

https://www.rapidtables.com/web/html/html-codes.html

Reply:
couldn't solve this section